pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -99,6 +105,12 @@ func classificationSupported(config *config.Config) bool {
 	return kprobe.ClassificationSupported(config)
 }
 
+func skipIfGoTLSNotSupported(t *testing.T, ctx testContext) {
+	if !gotlstestutil.GoTLSSupported(t, ctx.extras["config"].(*config.Config)) {
+		t.Skip("GoTLS is not supported")
+	}
+}
```

Comment:
no need for `ctx.extras["config"].(*config.Config)`

The configuration we need in `GoTLSSupported` needs `EnableRuntimeCompiler` and `EnableCORE`. you can get both from calling `config.New()`